lib/smart_proxy_ansible/plugin.rb

```diff
@@ -4,7 +4,17 @@ class Plugin < Proxy::Plugin
     https_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
 
     settings_file "ansible.yml"
-    default_settings :ansible_working_dir => '~/.foreman-ansible'
     plugin :ansible, Proxy::Ansible::VERSION
+
+    after_activation do
+      begin
+        require 'smart_proxy_dynflow_core'
+        require 'foreman_ansible_core'
+        ForemanAnsibleCore.initialize_settings(Proxy::Ansible::Plugin.settings.to_h)
+      rescue LoadError => _
+        # Dynflow core is not available in the proxy, will be handled
```

Comment:
The problem might be with `ForemanAnsibleCore.initialize_settings(Proxy::Ansible::Plugin.settings.to_h)` not getting called or failing… I have tried a fresh proxy setup with this https://gist.github.com/iNecas/dcc21b1eccce751b75baacf8e21b1b25 and was still not able to reproduce. I tried also ruby-2.3.0-preview1, still worked fine. Maybe something in logs?